packages/client/src/Call.ts

```diff
@@ -600,7 +602,7 @@ export class Call {
       this.clientStore.registerCall(this);
     }
 
-    this.applyDeviceConfig();
+    await this.applyDeviceConfig();
```

Comment:
Do we need `await` here if the `applyDeviceConfig` now catches the error?

Reply:
I was thinking about whether to add an `await` here or not. Eventually, I added it because it keeps the linear execution flow. There wasn't any other reason for the `await`.

Reply:
I'm fine with this change, I just was thinking if I'm missing something